programmers/큐.js

```diff
@@ -0,0 +1,101 @@
+class Queue {
+  constructor() {
+    this.data = [];
+    this.rear = 0;
+    this.size = 10000;
+  }
+
+  enqueue(element) {
+    if (this.rear < this.size) {
+      this.data[this.rear] = element;
+      this.rear = this.rear + 1;
+      return true;
+    }
+    // if there is no place to insert data
+    return false;
+  }
+
+  dequeue() {
+    if (this.isEmpty() === false) {
+      this.rear = this.rear - 1;
+      return this.data.shift();
+    }
+
+    return -1;
+  }
+
+  length() {
+    return this.rear;
+  }
+
+  isEmpty() {
```

Comment:
문제에서 지정된 명령어들이 있으니 클래스명을 이렇게 지정된 명령어로 작성하는게 가독성이 더 좋을거같아요!
(size, front, back도요)
```suggestion
  empty() {
```